src/jvm/test/resources/expected_logs/thread_creation_representation_test.txt

```diff
@@ -0,0 +1,65 @@
+|                                                                                                                                                   |           a ➜ 1 at ThreadCreationRepresentationTest.callMe(ThreadCreationRepresentationTest.kt:34)                                                                                         |                                                                                                                                                                                            |                                                                                                                                                                                            |
+|                                                                                                                                                   |           a = 2 at ThreadCreationRepresentationTest.callMe(ThreadCreationRepresentationTest.kt:34)                                                                                         |                                                                                                                                                                                            |                                                                                                                                                                                            |
+|                                                                                                                                                   |   result: void                                                                                                                                                                             |                                                                                                                                                                                            |                                                                                                                                                                                            |
+|                                                                                                                                                   |                                                                                                                                                                                            |                                                                                                                                                                                            | run()                                                                                                                                                                                      |
```

Comment:
Let's raise an issue to show `result: ...` only for actor calls (and solve it) -- we do not want this garbage in the trace :)

Reply:
#536